src/module/Accounting/config/module.config.php

```diff
@@ -42,9 +42,37 @@
 			),
 		),
 	),
+	
+	'bjyauthorize'=> array(
+		'resource_providers' => array(
+			'BjyAuthorize\Provider\Resource\Config' => array(
+				'Ora\Account' => array(),				
+			),
+		),
+		'rule_providers' => array(
+			'BjyAuthorize\Provider\Rule\Config' => array(
+				'allow' => array(
+					array(
+						array('user'), 
+						'Ora\Account', 
+						array('Accounting.Account.deposit'),
+						'Accounting\AccountHolderAssertion'), 
+					array(
+						array('user'), 
```

Comment:
La possibilità di accedere all'estratto conto di un conto corrente (_statement_) si differenzia in questo modo: se è un `OrganizationAccount`, tutti i membri dell'organizzazione lo possono vedere; se è un `Account` solo gli _Account Holders_